cognite/client/_api/datapoints.py

```diff
@@ -179,21 +180,26 @@ def _finalize_tasks(self, ordered_results: list[BaseConcurrentTask], resource_ls
             cognite_client=self.dps_client._cognite_client,
         )
 
-    def _make_dps_request_using_protobuf(self, payload: DatapointsPayload) -> bytes:
+    def _make_dps_request_using_protobuf(self, payload: DatapointsPayload, api_subversion: str | None) -> bytes:
         return self.dps_client._do_request(
             json=payload,
             method="POST",
             url_path=f"{self.dps_client._RESOURCE_PATH}/list",
             accept="application/protobuf",
             timeout=self.dps_client._config.timeout,
+            api_subversion=api_subversion,
```

Comment:
We pass `api_subversion` via several functions; should we set it on the instance instead to simplify?

Reply:
Also fixed in 9b6accd.